Tuplets skipped by Alt+Left and Alt+Right

brandonDavis264 opened the pull request:
Resolves: #17600

Tuplets skipped by Alt+Left and Alt+Right

On the forum I learned that the nextElement() function was the problem in the code with the EngravingItem namespace. The Engraving Item namespace include musical items seen on a bar line like a tuplet (what I need), tempo, or even notes. When clicking alt-left and right this function is called, so in the code I create a function in the Tuplet class that overrides the nextElment() function that is inherited from the Engraving item class. Also in the edge case of having truplet-ized rests the if statement to check if the item was a tuplet was also important. Also, when inspecting the .h file of the engravingItem we can also see the function nextSegementElement() which is used for navigation; adding a tuplet case statement also helps fix this issue. The same process is done for previous elements.

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

cbjeukendrup commented:
Hi! It looks like you opened this PR with the source and target repo the wrong way around. I somehow came across this by coincidence, but normally nobody would have discovered this. You're basically sending the changes from `musescore/MuseScore` into your own repo, instead of the other way around.

Instead, you should create a new branch in your own repository, make the changes on that branch, and then create a PR in the `musescore/MuseScore` repository.

cbjeukendrup commented:
Additionally, it looks like you haven't made any changes at all. Maybe you only made the changes on your own computer, and forgot to push them to your fork on GitHub? 
If you didn't intend to make changes, I don't know why this Pull Request is linked to https://github.com/musescore/MuseScore/issues/17600.

mike-spa committed:
Stave sharing

Eism committed:
fixed #32366: increased the margin for resizing area

miiizen committed:
Add tests

Use mscx so we can compare the inbuilt style defaults

miiizen committed:
Unlink the offset property from style values

miiizen committed:
Apply pos above/below styles to fermata

miiizen committed:
Add posAbove/Below styles during textbase layout

miiizen committed:
Add posAbove/Below styles during textline layout

miiizen committed:
Migrate offset properties

miiizen committed:
Fix measure number offset

miiizen committed:
Move defaultOffset and offsetSid to engravingItem

miiizen committed:
Add posAbove/Below styles during trill line layout

miiizen committed:
Migrate offset in <400 files

miiizen committed:
Remove code resetting offset to new styled position

Offsets which used to be "styled" are now just zero

miiizen committed:
Remove code referencing offset's styled value

Offset is not styled and its default is zero

miiizen committed:
Fix tempo line & rehearsal mark placement

TODO: this is being calculated at a bad time - before layout of the line, so we have no shape. The best we can do is use the info we already have (offset, min distance, staffheight) to check a single point. Previously, we were only checking offset

miiizen committed:
Migrate items with voice assignment properties after reading whole score

miiizen committed:
Refactor offset in text styles

This is now decoupled from the property value and can have values for above and below. Currently, where two values are available, only the posAbove value is used in the style dialog. However, now it will be easy to expose posBelow in the text styles menu as and when we want to.

miiizen committed:
Update tests

miiizen committed:
Rename defaultOffset to defaultPos

miiizen committed:
Add single function to get compat adjusted offset

miiizen committed:
Fix syncing chord & fret symbol vertical align

RomanPudashkin committed:
Use "System default" when switching between API

RomanPudashkin committed:
Show warning when no output devices are available

+ minor optimization

CubikingChill committed:
VST: defer IMidiMapping query to first access

CubikingChill committed:
VST: call setComponentState() after plugin init

CubikingChill committed:
VST: move component deactivation to main thread

miiizen committed:
Add spanner segments to spanners before reading

We rely on the context of the parent spanner when reading offsets.

miiizen committed:
Use undoChangeProperty

CubikingChill committed:
VST: clear plugin config when switching plugins

luapmartin committed:
add ThemeInfo conversion test scenarios

CubikingChill committed:
VST: skip setState when plugin returns empty state buffer

igorkorsukov committed:
[audio] removed clock

igorkorsukov committed:
[audio] moved iaudioengine to internal

mathesoncalum committed:
Use NSFloatingWindowLevel for plugins and extensions...
Instead of attaching/detaching parent and child windows.

kryksyh committed:
Merge pull request #33021 from luapmartin/luapmartin/fix-theme-extras

Fix theme extra colors lost after user customization

igorkorsukov committed:
[audio] added audio context

igorkorsukov committed:
fixed audio export

saintmatthieu committed:
TableSortFilterProxyModel

saintmatthieu committed:
Add sort-filter to table in UI gallery

saintmatthieu committed:
Make `TestTableViewModel` cells editable

saintmatthieu committed:
TableSortFilterProxyModel: list columns can also be used for sort

RomanPudashkin committed:
VST: fix invalid iterators caused by config modification in rescanParams()

mike-spa committed:
update utests

kryksyh committed:
Delete api creators on shutdown

kryksyh committed:
Merge pull request #33082 from kryksyh/delete_api_creators

Delete api creators on shutdown

igorkorsukov committed:
[audio] removed worker mode

RomanPudashkin committed:
Fix #33014: update state only when active

saintmatthieu committed:
Fix doubleinputvalidator for large values (10⁴ and more)

CubikingChill committed:
VST: rewind stream before setComponentState in state restore

igorkorsukov committed:
[audio] fixed track buffers allocation on each process call

RomanPudashkin committed:
Fix #33089: add missing trackChanges

mike-spa committed:
Fix read after free on bracket edit

The old comment said "Brackets may be deleted and replaced", but that is true also for systems (they get deleted during layout) so we can't store a system pointer and reuse it. We can only try to restore the selection by storing their indices.

mike-spa committed:
Fix read after free on broken spanners

Broken spanners were removed during read but they were kept in the EID register and could be referenced again.

RomanPudashkin committed:
Fix #32975: use notationPlayback()->trackAdded() / trackRemoved() as the single source of truth

Prevent accidental addition of duplicate tracks

igorkorsukov committed:
[audio] added audio node factory

mathesoncalum committed:
Merge pull request #33098 from mathesoncalum/33033-plugin_screens_bug

Fix #33033: Use NSFloatingWindowLevel instead of attaching/detaching parent and child windows [Master port]

saintmatthieu committed:
Reproduce the error with unit-test

saintmatthieu committed:
Add test case for small ranges

igorkorsukov committed:
[audio] fixed second export

CubikingChill committed:
Fix save selection corruption

miiizen committed:
Migrate items after layout

igorkorsukov committed:
[audio] simplified the track structure

RomanPudashkin committed:
VST: ensure rescanParams() is called only after the plugin is fully loaded

Avoid accidental configuration loss during initial controller–component state synchronization
(see syncControllerToComponentState)

RomanPudashkin committed:
VST: improve error handling when getting and setting state

igorkorsukov committed:
[audio] moved engine playback funcs to audio context

igorkorsukov committed:
[audio] changed isActive to mode

RomanPudashkin committed:
Merge pull request #33104 from CubikingChill/save-selection

Fix save selection corruption

igorkorsukov committed:
[audio] renamed RenderMode to ProcessMode

silasvorne committed:
Resolve https://github.com/musescore/MuseScore/pull/33112

Check pointer before de-reference to prevent crash.

igorkorsukov committed:
[audio] replaced seek(sec) to seek(TimePosition)

RomanPudashkin committed:
VST: fix blank view when moving/resizing on Windows

mvanhorn committed:
fix(engraving): correct Q_PROPERTY READ for markIrregularMeasures

The api/v1 Score Q_PROPERTY for markIrregularMeasures was reading from
showSoundFlags instead of its own getter:

    Q_PROPERTY(bool markIrregularMeasures READ showSoundFlags ...)

This meant QML plugins querying score.markIrregularMeasures received
the value of showSoundFlags instead. The apiv1::Score class already
declares the correct getter (markIrregularMeasures() at line 516),
so the fix is simply to point the READ at it.

Fixes #33121

mike-spa committed:
Fix linked stave visibility

Resolves: #33077

Reverts a change made in #32185

igorkorsukov committed:
[audio] audio RPС improvements

mike-spa committed:
corrections

mathesoncalum committed:
Merge pull request #33123 from mvanhorn/osc/33121-markirregular-typo

fix(engraving): correct Q_PROPERTY READ for markIrregularMeasures

igorkorsukov committed:
[audio] moved TimePosition to separate file

ajuncosa committed:
Fix check for language updates with 'system default' language

igorkorsukov committed:
[audio] fixed context rpc channel implementation

DmitryArefiev committed:
Merge pull request #33128 from ajuncosa/fix_32218

Fix check for language updates with 'system default' language

miiizen committed:
Move property setting to undoAddElement

saintmatthieu committed:
Third header click restores unsorted state

mike-spa committed:
Update loading screen

igorkorsukov committed:
[audio] added audio context init/deinit

igorkorsukov committed:
[audio] fixed set psotion in synths

mike-spa committed:
Remove loading message

saintmatthieu committed:
sort-order icon

RomanPudashkin committed:
Merge pull request #32841 from CubikingChill/roland-cloud-vst

Fix VST Crashes

github-actions[bot] committed:
Update in-repo translations from Transifex

github-actions[bot] committed:
Run `lupdate` with arguments `-no-obsolete`

alexpavlov96 committed:
not updating dead notes for transposition

alexpavlov96 committed:
added tests for checking fretting after transposition

alexpavlov96 committed:
setting critical color if printing isn't enabled

RomanPudashkin committed:
Fix #33109: fix crash when CLI finishes before audio engine is fully initialized

Partial initialization occurs in th_setupEngine(), but full initialization
never completes because rpcChannel->process() is not called (the RPC ticker depends on
the Qt event loop, which doesn't get a chance to process events), so
EngineRunning is never received

Ensure rpcChannel->process() is called at least once and always send EngineDeinit
to handle this case

miiizen committed:
Fix unity build

alexpavlov96 committed:
Merge pull request #33143 from alexpavlov96/negative_frets_color

setting critical color if printing isn't enabled

mathesoncalum committed:
Fix incorrect displaying of VSTs when dragging between screens (OS - Windows)

igorkorsukov committed:
[audio] reimplemented pass ctx in audio rpc

embarc-gabriel committed:
Narrow m_mutex scope to avoid deadlock while holding thdata->mutex

mike-spa committed:
Correct image size

RomanPudashkin committed:
VST: remove min & max buttons on Windows

mike-spa committed:
Merge pull request #33125 from mike-spa/fixLinkedStaveVisibility

Fix linked stave visibility

miiizen committed:
Fix dynamics style value

miiizen committed:
Merge pull request #33146 from miiizen/fixUnityBuild

Fix unity build

miiizen committed:
Merge pull request #32876 from miiizen/32859-harmonyAlignSync

Fix syncing chord & fret symbol vertical align

miiizen committed:
Merge pull request #32770 from miiizen/22622-offsetRefactor

Refactor offsets

RomanPudashkin committed:
Fix #33153: handle link activation in MouseArea instead of relying on Text.onLinkActivated

rettinghaus committed:
import tstamp to tick

silasvorne committed:
Delete the duplicate selection logic.

The logic below has already been executed above.
This causes the grace notes from anchor to chord to be selected twice.
After deleting them twice, the system enters an error state.
In this state, when closing the score, the same object gets deleted twice, and weird things happen during undo.

ajuncosa committed:
Fix input by duration getting stuck when arriving at the same note

RomanPudashkin committed:
Merge pull request #33163 from ajuncosa/fix_27640

Fix input by duration getting stuck when arriving at the same note

mathesoncalum committed:
Don't set device sample rate when switching audio API

kryksyh committed:
Add singleinstance handler, that prevents user from opening multiple instances in singleprocess mode

rettinghaus committed:
react on review

RomanPudashkin committed:
Revert "Fix VST plugin window crash on macOS when closing"

This reverts commit 3c6133b9cd801b97289597c1e2d231aeb3eb5e15.

RomanPudashkin committed:
Fix #32742: deinit VST view while NSView still exists

The crash occurred when closing VST plugin windows (e.g., ARIA) on macOS.
The issue was that IPlugView::removed() was being called during window
closure, which tries to access NSView objects that macOS may already be
deallocating as part of its own cleanup sequence.

On macOS, the Cocoa framework handles NSView cleanup automatically during
window destruction. Calling removed() at this point causes a crash because
it attempts to interact with partially or fully deallocated NSView objects.

On Windows and Linux, the underlying window handles (HWND/X11) remain valid
throughout the close event, so calling removed() is safe and necessary for
proper plugin cleanup.

mike-spa committed:
Merge pull request #33165 from mike-spa/port

Port engraving PRs to master

RomanPudashkin committed:
Audio API -> Audio driver

alexpavlov96 committed:
added engraving config to avoid transposing dead notes

igorkorsukov committed:
[audio] added responce to all rpc requests

RomanPudashkin committed:
Merge pull request #33175 from RomanPudashkin/rename_audio_api_to_audio_driver

Rename Audio API -> Audio driver

alexpavlov96 committed:
Merge pull request #32856 from alexpavlov96/not_updating_deadnotes

not updating dead notes for transposition

kryksyh committed:
Merge pull request #33172 from kryksyh/multiwindows-singleinstance

Add singleinstance handler

ajuncosa committed:
Fix arrange order under/overflow when pushing to the back/front

miiizen committed:
New Read500 module

igorkorsukov committed:
fixed fluid seq expressionLevel

RomanPudashkin committed:
Merge pull request #33180 from RomanPudashkin/port_fixes

Port fixes

RomanPudashkin committed:
Re-add screenChanged connection; timer-only approach is also unreliable

igorkorsukov committed:
[audio] added super mixer

igorkorsukov committed:
[audio] renamed engineplayer to contextplayer

alexpavlov96 committed:
same-string transpose: negative fret support

miiizen committed:
Remove 500 filetype implementations from 460

mike-spa committed:
Move layoutMeasure out of getNextMeasure

igorkorsukov committed:
[audio] removed iaudiosource from isynthesizer

igorkorsukov committed:
[win] fixed installer auto launch

igorkorsukov committed:
turn on multiwindows single proc mode by default

mathesoncalum committed:
Merge pull request #33164 from silasvorne/fix-duplicate-selection

Delete the duplicate selection logic.

RomanPudashkin committed:
Merge pull request #33192 from igorkorsukov/w/win/bug_33083

[win] fixed installer auto launch

miiizen committed:
Merge pull request #33183 from miiizen/500reader

New Read500 module

miiizen committed:
Serialize remaining enums with text

igorkorsukov committed:
switch coderabbit profile to chill

miiizen committed:
Remove read/write of action icon subtype

Palettes are versioned now, so this is safe

miiizen committed:
Update tests

mike-spa committed:
Add needLayout flag to measure

Calling layoutMeasure at the start of the while in SystemLayout::collectSystem could cause is to be laid out twice: once as last measure of the preceding system, once as the first measure of the next system (after the line break has happened). This flag resolves that, but it's also useful to have in general.

RomanPudashkin committed:
Merge pull request #33181 from RomanPudashkin/port_vst_fixes

Port VST fixes

igorkorsukov committed:
[audio] transformed eventaudiosource to eventaudionode

ajuncosa committed:
Fix 33194: creation of thumbnail preview when a score is created through 'Save selection'

(cherry picked from commit caf281436f8c4a725ed4821e656f5de5b2b8a746)

mike-spa committed:
Merge pull request #33189 from mike-spa/layoutMeasureRefactor

Move layoutMeasure out of getNextMeasure

dozzzzer committed:
TextInputArea: emit escaped() before focus loss so consumers can distinguish cancel from normal focus change

RomanPudashkin committed:
Fix GetInputParams

mike-spa committed:
Merge pull request #33132 from mike-spa/updateLoadingScreen

Update loading screen

igorkorsukov committed:
[audio] added fxnode

mathesoncalum committed:
Fix mixer not loading when switching pages

RomanPudashkin committed:
Merge pull request #33208 from RomanPudashkin/fix_GetInputParams

Fix GetInputParams

kryksyh committed:
Merge pull request #33207 from dozzzzer/fix/textinputarea-escaped-signal

TextInputArea: emit escaped() before focus loss

mathesoncalum committed:
New welcome dialog content for 4.7

kryksyh committed:
optional: remove max sort keys

kryksyh committed:
make cell responsible for providing sort value rather than proxy model

kryksyh committed:
simplify sort column selection

igorkorsukov committed:
[audio] added control node

kryksyh committed:
more rows and wider table for sort testing

ajuncosa committed:
Update About dialog

(cherry picked from commit 8ae83728b0b2dd913ea895ff195fca3382aae80b)

RomanPudashkin committed:
Fix #33088: fix ioc issues in TimeSignaturePropertiesDialog

RomanPudashkin committed:
Fix crash in CustomizeKitDialog: loadPitchesList() depends on notation

RomanPudashkin committed:
Welcome dialog: update link

igorkorsukov committed:
[audio] added signal node

RomanPudashkin committed:
Merge pull request #33220 from ajuncosa/fix-33194

Fix creation of thumbnail when a score is created via 'Save selection'

RomanPudashkin committed:
Merge pull request #33218 from ajuncosa/update_about

Update About dialog

igorkorsukov committed:
[audio] added playhead node

igorkorsukov committed:
[audio] optimized the control node

kryksyh committed:
fix crash on window close

During window close, some Injects can outlive their ModulesContextIoC because their owning QObjects are torn down via deleteLater() and the actual destruction runs on a later event-loop turn

igorkorsukov committed:
[audio] optimized the signal node

ajuncosa committed:
Update source files' headers

(cherry picked from commit d6dc6679e9f9e2825bda5cd9bf045ae485b8d800)

saintmatthieu committed:
fix warnings in demo StyledTableView

saintmatthieu committed:
fix: toggling sort with keyboard doesn't mess focus up

kryksyh committed:
Merge pull request #33225 from kryksyh/fix_crash_on_window_close

fix crash on window close

miiizen committed:
Merge pull request #33233 from mathesoncalum/master-ports-6

Porting 4.7 PRs to master (6)

github-actions[bot] committed:
Update in-repo translations from Transifex

github-actions[bot] committed:
Run `lupdate` with arguments `-no-obsolete`

alexpavlov96 committed:
negative frets conflict resolution

alexpavlov96 committed:
same-string transpose respects capo per string

RomanPudashkin committed:
Merge pull request #33245 from ajuncosa/update_headers

Update source files' headers

igorkorsukov committed:
[audio] added node tags

igorkorsukov committed:
[audio] added node dump

RomanPudashkin committed:
Wait until the progress dialog closes to prevent it from becoming the parent of subsequent dialogs

This prevents subsequent dialogs from being closed immediately after they appear

alexpavlov96 committed:
Merge pull request #33203 from alexpavlov96/same_string_fretting

same-string transpose: negative fret support

alexpavlov96 committed:
removing accidentals from dead note, if it shouldn't be tranposed

RomanPudashkin committed:
Merge pull request #33246 from RomanPudashkin/port_fixes

Port fixes

alexpavlov96 committed:
Merge pull request #33250 from alexpavlov96/deadnote_strip_accidentails

removing accidentals from dead note, if it shouldn't be tranposed

igorkorsukov committed:
[audio] added fx chain

igorkorsukov committed:
[audio] replaced mixchannel on trackchain

igorkorsukov committed:
[audio] replaced mixchannel on trackchain

RomanPudashkin committed:
Merge pull request #33168 from rettinghaus/mei/segments

[MEI] Import floating objects

igorkorsukov committed:
[audio] added master track chain

mike-spa committed:
Palette compat chord brackets

igorkorsukov committed:
[audio] fixed signal node

rettinghaus committed:
i/o line arrows

rettinghaus committed:
fix duplicated export

igorkorsukov committed:
[audio] clean up mixer

igorkorsukov committed:
[audio] fixed fx chain

DmitryArefiev committed:
Merge pull request #33258 from mike-spa/paletteCompatChordBrackets

Palette compat chord brackets

miiizen committed:
Merge pull request #33260 from rettinghaus/xml/arrow

MusicXML: add support for line arrows

mike-spa committed:
Merge pull request #33197 from miiizen/serializeEnums

Serialize all enums as text when writing to file

miiizen committed:
Merge pull request #33022 from miiizen/moveNoteDotLineFunc

Move setDotRelativeLine to ChordLayout

mike-spa committed:
Loading screen

RomanPudashkin committed:
Merge pull request #33269 from mike-spa/loadingScreenMaster

Loading screen

ajuncosa committed:
Fix playback from timeline elements

igorkorsukov committed:
[audio] split the track parameters into parts

mathesoncalum committed:
Standardize sizing of welcome dialog images

ajuncosa committed:
Fix 'Save selection' bug due to stale pointers in EID register

mathesoncalum committed:
Welcome dialog - update YouTube link

RomanPudashkin committed:
Fix #33142: fix ioc issues in SpecialCharactersDialog

mike-spa committed:
Merge pull request #33276 from ajuncosa/fix-save-selection

Fix 'Save selection' bug due to stale pointers in EID register

RomanPudashkin committed:
Remove legacy percussion panel

igorkorsukov committed:
[audio] added exec operation to player

silasvorne committed:
Fix "show actions list" crash

The crash originates from the diagnostics module, so I didn't report a bug. `iocCtxForQmlObject` should capture `this` instead of `parent`.

igorkorsukov committed:
[audio] added ONLY_ON_OPERATION_EXEC

kryksyh committed:
Merge pull request #33036 from saintmatthieu/allow-empty-table-views

TableSortFilterProxyModel

mathesoncalum committed:
Fix crash when loading palettes from older MSS version

Jojo-Schmitz committed:
Fix MSVC compiler warnings

reg.: 'initializing': conversion from 'int' to 'kors::modularity::IoCID', signed/unsigned mismatch (C4245)

Jojo-Schmitz committed:
Fix MSVC compiler warnings

reg.: declaration of 'ret' hides previous local declaration (C4456)
and: declaration of 'item' hides previous local declaration (C4456)

Jojo-Schmitz committed:
Fix MSVC compiler warning

reg.: 'pitchOffset': unreferenced parameter (C4100)

Jojo-Schmitz committed:
Fix MSVC compiler warnings

reg.: '=': conversion from 'size_t' to 'int', possible loss of data (C4267)

RomanPudashkin committed:
Fix #33288: avoid destroying QML objects during context menu signal handling

The handler may close the current score and destroy docks before the QML callback completes

DmitryArefiev committed:
Merge pull request #33178 from ajuncosa/fix_27771

Fix arrange order under/overflow when pushing to the back/front

mathesoncalum committed:
Merge pull request #33264 from rettinghaus/xml/gracetext

Fix #32851: duplicated export

RomanPudashkin committed:
Merge pull request #33287 from RomanPudashkin/remove_legacy_percussion_panel

Remove legacy percussion panel

igorkorsukov committed:
the muse framework is now a submodule

mathesoncalum committed:
Merge pull request #33296 from Jojo-Schmitz/compiler-warnings

Fix MSVC compiler warnings

mike-spa committed:
code review

igorkorsukov committed:
adjust muse framework

Jojo-Schmitz committed:
Fix build

DmitryArefiev committed:
Merge pull request #33263 from ajuncosa/fix-32613

Fix playback from timeline elements

igorkorsukov committed:
[ci] added check submodules

mike-spa committed:
Merge pull request #33080 from mike-spa/STAVE_SHARING

Stave sharing - Part 0

Jojo-Schmitz committed:
Fix MSVC compiler warning

reg.: 'keyList': local variable is initialized but not referenced (C4189)

kryksyh committed:
Workflow changes to allow workflow reuse in the framework repo: https://github.com/musescore/MuseFramework/pull/8

cbjeukendrup committed:
Move Home > Scores > Cloud scores components into project module

Near all other Home > Scores components

RomanPudashkin committed:
Merge pull request #33316 from Jojo-Schmitz/compiler-warnings

Fix MSVC compiler warning

RomanPudashkin committed:
Fix #33315: nextCR(track) can return MeasureRepeat since it's derived from ChordRest

igorkorsukov committed:
bump MF submodule

igorkorsukov committed:
[ci] now the submodule check checks for strict equality

igorkorsukov committed:
bump MF submodul

rettinghaus committed:
add missed cases

RomanPudashkin committed:
Merge pull request #33336 from rettinghaus/fix/warning

Fix compiler warnings

kryksyh committed:
Merge pull request #33326 from kryksyh/master

Workflow changes to allow workflow reuse in the framework repo

RomanPudashkin committed:
Merge pull request #33338 from RomanPudashkin/port_fixes

Port fixes

RomanPudashkin committed:
Merge pull request #33217 from RomanPudashkin/fix_time_sig_dialog_crash

Fix #33088: Time signature properties crash

igorkorsukov committed:
bump MF submodule

igorkorsukov committed:
bump MF submodule
